modules/appeals_api/app/controllers/appeals_api/v2/decision_reviews/notice_of_disagreements_controller.rb

```diff
@@ -44,16 +44,13 @@ def validate
   end
 
   def schema
-    # TODO: Return full schema after we've validated all Non-Veteran Claimant functionality
     response = AppealsApi::JsonSchemaToSwaggerConverter.remove_comments(
       AppealsApi::FormSchemas.new(
         SCHEMA_ERROR_TYPE,
         schema_version: 'v2'
       ).schema(self.class::FORM_NUMBER)
     )
-    response.tap do |s|
```

Comment:
I think we want to keep this filtering in, since we don't currently want our production schemas to include NVC information for NODv2.

Reply:
You're so right. I'll get that back in!